### ISSUE TITLE
Is it possible to disable hotkey for pathfinding while typing in terminal?

### ISSUE THREAD

**Issue by Yakka-2k** [NilaierMusic/NavigatingStars#2]
Love the mod! So helpful and I use it all the time!

Could it be possible to disable the pathfinding hotkey while typing in the terminal?
Alternatively, could there be a way to disable the warning pop-up (or sound) when no path is found?

When I'm typing in the terminal, there are numerous times I type my hotkey, and the constant warning popup and sound become a little too much at times.

**Comment by NilaierMusic** [comment]
Yeah, i'll implement this when i have some free time. For now, you can just put the hotkey on something that isn't used in the terminal (for example, backslash, forward slash, punctuation symbols and etc.)
